docs/source/sci_api_tutorial/Structure.rst

```diff
@@ -0,0 +1,113 @@
+==============================================
+Structure
+==============================================
+
+Briefs
+==============================================
+This class define the core data structure of EnzyHTP: Structure. Structure stands for the single enzyme structure.
+As the core data class, Structure will be solely designed for **storing, accessing and editing** structural data.
+
+Input/Output
+==============================================
+
+.. panels::
+
+    :column: col-lg-12 col-md-12 col-sm-12 col-xs-12 p-2 text-left
+
+    .. image:: ../../figures/Structure_io.svg
+        :width: 100%
+        :alt: assign_mutant_io 
+

```

Comment:
Good job making the DFD! Under the same theory framework there is a type of diagram called the "class diagram" (https://www.lucidchart.com/pages/uml-class-diagram) Let's make that instead. We can still keep the arrow for the operations that can make the class. And we can replace the current Structure() block with something similar to the class box in the link (i.e., showing all the 1. attribute 2. property 3. methods).

Something advance we can do is to show the inter-class relationship such as Structure() carries Chain() but this might be too much for this PR.